pipeline/list-releases.bash

```diff
@@ -39,8 +39,8 @@ echo -e "Listing helm releases\n"
 
 "${bin_path}"/ck8s ops helmfile "${cluster_abbr}" status || true
 
-failed_releases=$("${bin_path}"/ck8s ops helmfile "${cluster_abbr}" status --args="--output=json" 2>/dev/null \
-    | jq 'select(.info.status=="failed")' || true)
```

Comment:
Wont this script fail if you remove this and there's no failed releases?

Reply:
https://github.com/elastisys/compliantkubernetes-apps/actions/runs/2312398279

I think this have somehow been handled by the fact that the script will fail anyway if a release is failing before this :thinking: